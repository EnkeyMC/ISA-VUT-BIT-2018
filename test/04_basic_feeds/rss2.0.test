NAME=Basic RSS 2.0 parsing
ARGS=http://www.mocky.io/v2/5bd3096a3400003600cfdff9
RC=0
CHECK_OUTPUT=1
