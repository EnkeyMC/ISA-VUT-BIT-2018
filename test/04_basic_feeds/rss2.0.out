*** Scripting News ***
Law and Order
Rule 1
Really early morning no-coffee notes
